pkg/api/kyverno/v1/policy_types.go

```diff
@@ -198,6 +198,8 @@ type MatchResources struct {
 	UserInfo `json:",omitempty" yaml:",omitempty"`
 
 	// ResourceDescription contains information about the resource being created or modified.
+	// Requires at least one resource to be specified when under MatchResources, if not,
+	// then ExcludeResources should be present instead.
```

Comment:
Can we change to "Requires at least one **tag** to be ..."

Reply:
@RinkiyaKeDad From this validation, it seems that `match` is required in the rule definition: https://github.com/kyverno/kyverno/blob/10c714d5ba735d72f5e0395004e72de06cf8e9ae/pkg/policy/validate.go#L477-L480

So we need to remove "if not, then ExcludeResources should be present instead.".

Reply:
@realshuting - Made the edit, please review.